extension-manifest-v2/src/background.js

```diff
@@ -100,24 +100,6 @@ function updateDBs() {
 
 			c2pDb.update(data.click2play);
 			compDb.update(data.compatibility);
-			bugDb.update(data.bugs, (result) => {
-				log('CHECK LIBRARY VERSION CALLED', result);
-				if (result.success) {
-					const nowTime = Date.now();
-					conf.bugs_last_checked = nowTime;
-					if (result.updated) {
-						log('BUGS LAST UPDATED UPDATED', new Date());
-						conf.bugs_last_updated = nowTime;
-					}
-				}
-				resolve({
```

Comment:
Won't removing that "resolve" leave the Promise that "updateDBs" returns dangling?

Reply:
good catch